### ISSUE TITLE
Get photos

### ISSUE THREAD

**Issue by GLivshits** [kevinzg/facebook-scraper#308]
Hello. For me it seems like get_photos function does not work at all. It always returns an empty iterator, even if there are lots of photos in scraped account. I use credentials tuple to login.

**Comment by neon-ninja** [comment]
Working fine for me, what page or profile are you having this problem with?

**Comment by GLivshits** [comment]
For example, this user: urve.ringmets
However, I was able to get the right JSON with links from account in the later issue by fashandatafields.

**Comment by lennoxho** [comment]
Hi! I contributed the get_photos() API.
I would describe this as a missing feature.

This is the page "format" get_photos() is expecting:
https://m.facebook.com/Nintendo/photos/
(We are scraping the "All Photos" section at the bottom)

But check out the photos page for the user urve.ringmets
https://m.facebook.com/urve.ringmets/photos

@neon-ninja correct me if I'm wrong, but the way things are put together right now, it's not pretty to do 2-level page iterations, no?

Let me rephrase. This user's photo page is scrollable (it has links to albums), but then each album page is itself scrollable.
I don't think we are well equipped to handle this right now?

**Comment by neon-ninja** [comment]
Should be doable, it's the same sort of concept as yielding pages and then yielding posts on each page. You'd probably want a field to indicate what photos belong to what albums
